src/dialog/overflow-menu/overflow-menu-pane.component.ts

```diff
@@ -46,12 +46,20 @@ export class OverflowMenuPane extends Dialog {
 			this.dialogConfig.menuLabel = this.i18n.get().OVERFLOW_MENU.OVERFLOW;
 		}
 
-		setTimeout(() => this.listItems()[0].focus());
+		setTimeout(() => {
+			let options = Array.from<any>(this.dialog.nativeElement.querySelectorAll("*"));
+
+			options.forEach(element => {
+				if (element.tabIndex === 0) {
```

Comment:
Maybe something like this? Per @zvonimirfras's comment
```suggestion
				if (element.tabIndex && element.tabIndex === 0) {
```